proto/warrior.proto

```diff
@@ -57,12 +57,62 @@ message WarriorTalents {
 
 message Warrior {
     message Rotation {
+		enum Type {
+			ArmsSlam = 0;
+			ArmsDW = 1;
+			Fury = 2;
+			General = 3;
```

Comment:
I'm assuming that the 'General' category is fields shared by all rotations? If so you dont need an enum and separate GeneralRotation message to hold them, just put them alongside the other fields in Rotation.

Reply:
Done in a21327c1.